<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="5000" />
  <arg name="chain_start" default="base_link" />
  <arg name="chain_end" default="moveo_link5" />
  <!--
  <arg name="chain_start" default="torso_lift_link" />
  <arg name="chain_end" default="r_wrist_roll_link" />
  -->
  <arg name="timeout" default="0.05" />
  <!--
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find pr2_description)/robots/pr2.urdf.xacro'" />
  -->
  <arg name="model" default="$(find moveo_urdf)/urdf/moveo_urdf.urdf"/>
  <param name="robot_description" textfile="$(arg model)" />
  
  <node name="moveo_moveit_move_group" pkg="moveo_moveit_move_group" type="ik_test" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
